### PR TITLE
[Pull Request] Struct optimized, code optimized.

### PR DESCRIPTION
**Pull request by Dynesshely**
## What does this PR do?

Features:

- Updated dependencies.

Refactorings:

- Use one dart file to export other dart files on same level.
- Use more extension methods.
- Apart some data and info to alone dart files.
- Apart some ui code to alone widgets in alone dart files.

## Related issues

None.

### CODE REVIEW

File: kitx_mobile/lib/utils/handlers/task_handler.dart
Comment (Cranyozen) [inline kitx_mobile/lib/utils/handlers/task_handler.dart:1]: Is this a 'handler'?
I think it's better to put this in /lib/utils/delay.dart

File: kitx_mobile/lib/pages/pages.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+/// Get Pages
+List<GetPage<dynamic>> getPages() => [
+      GetPage(name: '/', page: () => HomePage()),
+      GetPage(name: '/DevicePage/', page: () => DevicePage()),
+      GetPage(name: '/AccountPage/', page: () => AccountPage()),
+      GetPage(name: '/TestPage/', page: () => TestPage()),
+      GetPage(name: '/SettingsPage/', page: () => SettingsPage()),
+      GetPage(name: '/AboutPage/', page: () => AboutPage()),
+    ];
```
Comment (Cranyozen) [inline kitx_mobile/lib/pages/pages.dart]: Strange Indentation

**Review by Cranyozen — CHANGES_REQUESTED** [review]
I think it's better to refactor `int_ext.dart` and `theme_mode_ext.dart` to one file like enmus/theme_mode.dart or directly themes/convert.dart.
Because these two files are about theme mod, but not whole int class.
It's not necessary to expand whole int class.

File: kitx_mobile/lib/pages/pages.dart (hunk 1 above)
Comment (Dynesshely) [reply kitx_mobile/lib/pages/pages.dart]: Auto indentated by flutter itself.

File: kitx_mobile/lib/utils/handlers/task_handler.dart
Comment (Dynesshely) [reply kitx_mobile/lib/utils/handlers/task_handler.dart:1]: I am going to upgrade it to a task system, after that, the `delay` method will be deprecated.

**Comment by Dynesshely** [comment]
> I think it's better to refactor `int_ext.dart` and `theme_mode_ext.dart` to one file like enmus/theme_mode.dart or directly themes/convert.dart. Because these two files are about theme mod, but not whole int class. It's not necessary to expand whole int class.

You're right, I am going to think another way to achieve an elegant converting way.

**Comment by Dynesshely** [comment]
> > I think it's better to refactor `int_ext.dart` and `theme_mode_ext.dart` to one file like enmus/theme_mode.dart or directly themes/convert.dart. Because these two files are about theme mod, but not whole int class. It's not necessary to expand whole int class.
> 
> You're right, I am going to think another way to achieve an elegant converting way.

I didn't think of a better way, since the `int` and `ThemeMode` classes are not part of what we can modify, except for adding extension methods for them, I can't think of other codes that can elegantly implement the conversion between `int` and `ThemeMode`.

File: kitx_mobile/lib/utils/handlers/task_handler.dart
Comment (Dynesshely) [reply kitx_mobile/lib/utils/handlers/task_handler.dart:1]: In commit d4b447b, I convert delay function to alone class. This might solved problem.

**Review by Sadinary — APPROVED** [review]
Well done.

**Pull request merged**